draft-ietf-quic-tls.md

```diff
@@ -771,8 +771,7 @@ TLS 1.3 (see {{initial-secrets}}).
 ## Initial Secrets {#initial-secrets}
 
 Initial packets are protected with a secret derived from the Destination
-Connection ID field from the client's first Initial packet of the
-connection. Specifically:
+Connection ID field from the client's Initial packet. Specifically:
```

Comment:
Why this change, when it removes the precision about *which* Initial packet?

Reply:
I can't remember why I changed that, reverted.